client/src/pages/VotePage.tsx

```diff
@@ -111,7 +127,47 @@ export const VotePage = () => {
     <div className={"flex flex-col items-center px-5"}>
       <Header text={`Poll Code: ${pollCode}`} />
       <Header text={`Selected Option: ${selectedOption}`} />
-      <div className={"flex flex-col max-w-md"}>{optionButtons()}</div>
+      <div className={"relative"}>
+        <div className={"flex flex-col max-w-md"}>{optionButtons()}</div>
+        {loading ? (
+          <div className={"z-50 top-0 left-0 block w-full h-full absolute "}>
+            <div
+              className={
+                "flex items-center w-full bg- h-full flex-col justify-center align-middle"
+              }
+              style={{ backgroundColor: "rgba(0,0,0,0.5)" }}
+            >
+              <div role="status">
+                <svg
+                  aria-hidden="true"
+                  className="mr-2 w-8 h-8 text-gray-200 animate-spin dark:text-gray-900 fill-blue-600"
```

Comment:
This fixes it for me.

```suggestion
                  className="w-8 h-8 text-gray-200 animate-spin dark:text-gray-900 fill-blue-600"
```